bindings/python/tests/fdbcli_tests.py

```diff
@@ -440,6 +442,13 @@ def throttle(logger):
     assert enable_flag == "`0'"
     # TODO : test manual throttling, not easy to do now
 
+def wait_for_database_available(logger):
+    # sometimes the change takes some time to have effect and the database can be unavailable at that time
+    # this is to wait until the database is available again
+    while not get_value_from_status_json(True, 'client', 'database_status', 'available'):
+        time.sleep(1)
+        logger.debug("Database unavailable for now, wait for one second")
```

Comment:
nit: shouldn't this line be moved before the sleep() ?

Reply:
yeah, good catch.